src/trusted/service_runtime/sel_ldr.c

```diff
@@ -912,8 +933,9 @@ void NaClSetUpBootstrapChannel(struct NaClApp  *nap,
   hdr.ndescv = descs;
   hdr.ndesc_length = NACL_ARRAY_SIZE(descs);
 
-  rv = (*NACL_VTBL(NaClDesc, channel)->SendMsg)((struct NaClDesc *) channel,
-                                                &hdr, 0);
+  if (!channel)
```

Comment:
what can `NULL` channel between here and line 905?

Reply:
Ah, linter was giving me a false positive here about a `NULL` dereference. It also doesn't understand `LOG_FATAL` aborts :/. Will fix, good catch!